content/waku/2024-gantt.md

```diff
@@ -0,0 +1,180 @@
+        %% Estimate TBC
+        Review MVDS usage and fail path: crit, mvds, after review-conn-mgmt-1, 6w
+        %% Estimate/start TBC
+        (testing) direct messages: test-direct-msg, after mvds, 4w
+        %% Estimate TBC
+        (testing) Custom shard impl of Communities: test-custom-shard, after test-direct-msg, 4w 
+    section (chat) kaichaosun
+        %% Estimation TBC
+        Reliability Protocol for Relay (status-go): crit, rel-relay-g, 2024-05-13, 20w
+        %% TBC estimate
+        Multicast e2e reliability protocol Status integration: crit, m-e2e-rel-g, after m-e2e-rel-r, 10w
+        %% TBC estimate
+        (telemetry) Multicast message reliability: telem-m-msg-rel, after m-e2e-rel-g, 4w
+%% go-Waku/Chat/Eco Dev
+    section (gwak) richard-ramos
+        %% Dates and dogfooding TBC
+        Store v3 (go-waku client only): crit, storev3-g, 2024-02-01, 2024-04-12
+        %% Estimate TBD
+        DoS protection for req-res protocols (go-waku client only): crit, dosreqresg, after storev3-g, 4w
+        %% Estimate TBC - assuming parallel work possible
+        Review connection management: crit, review-conn-mgmt-2, 2024-05-13, 8w
+        %% Estimate TBC
+        (metric) Count store messages: count-store-msg, after review-conn-mgmt-2, 4w
+        %% TBC Estimate
+        Usage of rendezvous: crit, rendezvous-g, after mvds, 6w
+        %% TBC Estimate
+        Nwaku in Golang desktop: nwaku-golang-desk-g, after rendezvous-g, 2w
+        %% TBC Estimate
+        Nwaku in Golang - relay: nwaku-golang-relay-g, after nwaku-golang-desk-g, 4w
+        %% TBC Estimate
+        Nwaku in Status Desktop (Relay, *nix): nwaku-status-d-g, after nwaku-golang-relay-g, 8w
+    section (gwak) chaitanyaprem
+        Store v3 (sync): crit, 2024-02-08, 2024-04-26
+        %% Estimate TBC assuming no RFC here
+        Reliability Protocol for Resource-Restricted Clients (status-go): crit, rel-resreq-g, 2024-05-13, 10w
+        %% TBC Estimate
+        Sharding peer mgmt and discovery hardening (go-waku): crit, sh-peer-mgmt-g, after rel-resreq-g, 8w
+    section (eco dev) vpavlin
+        %% TBC estimate and start
+        (telemetry) direct message reliability: telem-d-msg-rel, 2024-06-17, 6w
+        %% TBC estimate and start
+        (telemetry) Measure Bandwidth: telem-bandwidth, after telem-d-msg-rel, 8w
+%% Research Team
+    section (rsrch) - jm-clius
+        White Paper: whitepaper, 2024-01-01, 2024-12-01
+        %% TBC end
+        RFC review: rfc-review, 2024-01-01, 90d
+        %% TBC start and estimation
+        Status Communities protocol scaling recommendation: status-proto-r, after rfc-review, 12w
+    section (rsrch) alrevuelta
+        %% TBC that work is done
+        RLN in resource-restricted clients: rlnrestricted, 2024-01-01, 110d
+        %% TBC estimation
+        RLNv2 in nwaku: rlnv2-n, 2024-02-01, 18w
+        Maturing RLN roadmap: maturing-rln-roadmap, 2024-01-01, 2024-12-01
```

Comment:
We should add a section for Tanya too. For now she will follow the same roadmap as alrevuelta as simulation engineer for the RLN items.